yt_dlp/extractor/svt.py

```diff
@@ -429,24 +469,27 @@ def suitable(cls, url):
     def _real_extract(self, url):
         path, display_id = self._match_valid_url(url).groups()
 
-        article = self._download_json(
-            'https://api.svt.se/nss-api/page/' + path, display_id,
-            query={'q': 'articles'})['articles']['content'][0]
-
+        webpage = self._download_webpage(url, display_id)
+        title = self._og_search_title(webpage)
         entries = []
 
-        def _process_content(content):
-            if content.get('_type') in ('VIDEOCLIP', 'VIDEOEPISODE'):
-                video_id = compat_str(content['image']['svtId'])
-                entries.append(self.url_result(
-                    'svt:' + video_id, SVTPlayIE.ie_key(), video_id))
+        urqlJson = self._search_json(r'window\.svt\.nyh\.urqlState[^{]+', webpage, 'video_id', display_id, default=None)
```

Comment:
- snake_case for var names
- we should use `\s*=` instead of `[^{]+`
- line too long
- why default to `None`? imo this should be fatal
```suggestion
        urql_state = self._search_json(
            r'window\.svt\.nyh\.urqlState\s*=', webpage, 'video id', display_id)
```